src/megaapi_impl.cpp

```diff
@@ -10536,8 +10536,8 @@ void MegaApiImpl::notify_confirmation(const char *email)
 
 void MegaApiImpl::notify_disconnect()
 {
-    MegaEventPrivate *event = new MegaEventPrivate(MegaEvent::EVENT_DISCONNECT);
-    fireOnEvent(event);
+    //MegaEventPrivate *event = new MegaEventPrivate(MegaEvent::EVENT_DISCONNECT);
+    //fireOnEvent(event);
```

Comment:
Once @alber2510 changes MEGAchat to not disconnect on this event if there's an on-going call, I think this callback is suitable and should be kept here.